requirements.txt

```diff
@@ -27,7 +27,7 @@ pytoml==0.1.19
 raiden-contracts==0.19.1
 raiden-webui==0.8.0
 requests==2.20.0
-structlog==18.2.0
+structlog==20.1.0
```

Comment:
I think some tests will fail with this change if i'm not wrong. I remember that we had some issue with that in the past.